src/Features/Core/Portable/UseExplicitTupleName/UseExplicitTupleNameDiagnosticAnalyzer.cs

```diff
@@ -37,35 +37,39 @@ private void AnalyzeOperation(OperationAnalysisContext context)
                 return;
             }
 
+            // We only create a diagnostic if the option's value is set to true.
             var option = optionSet.GetOption(CodeStyleOptions.PreferExplicitTupleNames, context.Compilation.Language);
-            var severity = option.Notification.Severity;
-            if (severity == ReportDiagnostic.Suppress)
+            if (option.Value)
             {
```

Comment:
much much simpler (for diff purposes) to just have:

```
if (!option.Value)
{
    return;
}
```

Reply:
That's a good point -- cleaned it up :)